build/rbac/keep-rbac-yaml.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+set -eEuo pipefail
+
+# READS FROM STDIN
+# WRITES TO STDOUT
+# DEBUGS TO STDERR
+
+: ${YQ:=yq}
+
+if [[ "$($YQ --version)" != "yq (https://github.com/mikefarah/yq/) version 4."* ]]; then
+  echo "yq must be version 4.x"
+  exit 1
+fi
+
+temp_dir="$(mktemp -d)"
+pushd "${temp_dir}" &>/dev/stderr
```

Comment:
Why not just add `-x` to the script instead of sending it to `/dev/stderr`?

Reply:
The output from `pushd` normally goes to stdout, which will mean any redirected output of the script will get stored in `rbac.yaml` or wherever it's sent to.